packages/espressocash_app/lib/app/screens/authenticated/authenticated_flow_screen.dart

```diff
@@ -70,17 +70,24 @@ class _AuthenticatedFlowScreenState extends State<AuthenticatedFlowScreen> {
                 ),
                 const ODPModule(),
                 const OSKPModule(),
-                const ISKPModule(),
                 const InvestmentModule(),
                 const ActivitiesModule(),
                 const IntercomModule(),
                 const FavoriteTokensModule(),
-                const ISLPModule(),
                 const SwapModule(),
                 const PopularTokensModule(),
                 OnboardingModule(mnemonic: mnemonic),
               ],
-              child: AutoRouter(key: _homeRouterKey),
+              child: AutoRouter(
+                key: _homeRouterKey,
+                builder: (context, child) => MultiProvider(
```

Comment:
Why should they only be for authenticated routes?

Reply:
They are moved down the router since now the incoming link listeners will push if there is a valid link